config/jobs/kubernetes/cloud-provider-gcp/cloud-provider-gcp-presubmits.yaml

```diff
@@ -28,9 +28,6 @@ presubmits:
     always_run: true
     decorate: true
     path_alias: k8s.io/cloud-provider-gcp
-    labels:
-      preset-service-account: "true"
-      preset-k8s-ssh: "true"
```

Comment:
theses presets are needed for the end to end job

Reply:
Updated. Thanks!